cla-frontend-contributor-console/.serverless_plugins/serverless-lambda-version/.npmignore

```diff
@@ -1,3 +1,6 @@
+# Copyright The Linux Foundation and each contributor to CommunityBridge.
```

Comment:
Same comment as above re: serverless-lambda-version -- these notices should not be added if these files originate from an upstream source and have not added content to them for CommunityBridge.